04-onion-routing.md

```diff
@@ -280,6 +283,9 @@ The writer:
       - MUST include `payment_data`
       - MUST set `payment_secret` to the one provided
       - MUST set `total_msat` to the total amount it will send
+    - if the recipient provided `payment_metadata`:
+      - MUST include `payment_metadata`
+      - MUST not apply an arbitrary limit to the size of `payment_metadata`.
```

Comment:
I mean I'd say "128 bytes" isn't "arbitrary" :). Maybe rephrase as "MUST not apply any limits to the size of `payment_metadata` except the limits implied by the fixed onion size" or so?

Reply:
Updated.